src/app/services/handler-toastr.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { ToastrService } from 'ngx-toastr';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class HandlerToastrService {
+
+  constructor(private toaster: ToastrService) { }
```

Comment:
Please fix inconsistency between names 'toaster' in not injected from 'ToasterService' but from 'ToastrService'.
That also related to your custom handler toastr service used in components.